src/features/groups/Edit/index.tsx

```diff
@@ -13,65 +13,76 @@ import Members from '../Members';
 import { ROUTES } from 'constants/routes';
 
 import { ReactComponent as BackIconSVG } from 'assets/images/back-icon.svg';
-import { ReactComponent as ArrowRightSVG } from 'assets/images/arrow-right.svg';
+import { SliderItemsWrapper, SliderNavItem } from './index.styled';
 
-import { GroupEditContentWrapper } from './index.styled';
-import Loader from 'components/Loader';
-import { ButtonBase, TextUnderline } from 'components';
-
-enum GroupEditMode {
-  GeneralInfo,
-  Members,
-}
+import Slider from "react-slick";
 
 const Edit = () => {
   const { t } = useTranslation('groups');
   const navigate = useNavigate();
   const { groupId } = useParams();
+  const sliderItems = groupId ? ["mainInfo", "members", "statistics"] : ["mainInfo"];
```

Comment:
Would it make sense to replace these strings with enum?

Reply:
Yeah, it definitely would be better